docs/content/preview/releases/techadvisories/ta-22935.md

```diff
@@ -0,0 +1,179 @@
+    CREATE DATABASE
+    ```
+
+1. Connect to database seqtest1 and create sequence foo in seqtest1. Allocate three values from foo by calling the function nextval three times as follows:
+
+    ```sql
+    \c seqtest1
+    ```
+
+    ```output
+    You are now connected to database "seqtest1" as user "yugabyte".
+    ```
+
+    ```sql
+    create sequence foo;
+    select oid, relname from pg_class where relkind = 'S';
+    ```
+
+    ```output
+      oid  | relname
+    -------+---------
+     16384 | foo
+    (1 row)
+    ```
+
+    ```sql
+    select nextval('foo');
+    ```
+
+    ```output
+     nextval
+    ---------
+           1
+    (1 row)
+    ```
+
+    ```sql
+    select nextval('foo');
+    ```
+
+    ```output
+    nextval
+    ---------
+           2
+    (1 row)
+    ```
+
+    ```sql
+    select nextval('foo');
+    ```
+
+    ```output
+    nextval
+    ---------
+           3
+    (1 row)
+    ```
+
+1. Connect to database seqtest2, create a new sequence bar in database seqtest2 and allocate value by calling the function nextval. In this scenario, sequence bar has the same OID as sequence foo, so it results in using the same cache on TServer. This result in nextval function returns 4 which is the next available sequence value in the cache. The correct behavior would be to return 1, as this is first time values are allocated from the sequence bar and default for an sequence minvalue is 1.
+
+    ```sql
+    \c seqtest2
+    seqtest2=# create sequence bar;
+    select oid, relname from pg_class where relkind = 'S';
+    ```
+
+    ```output
+      oid  | relname
+    -------+---------
+     16384 | foo
+    (1 row)
+    ```
+
+    ```sql
+    seqtest2=# select nextval('bar');
+    ```
+
+    ```output
+     nextval
+    ---------
+           4
+    (1 row)
+    ```
+
+    TServer sequence cache entries are never invalidated and reside in memory until TServer shuts down as described in the following steps.
+
+1. Drop the database seqtest1 and seqtest2 which drops all the objects including sequences foo and bar. Create a new database seqtest3 and connect to database seqtest3.
+
+    ```sql
+    drop database seqtest1;
+    drop database seqtest2;
+    create database seqtest3;
+    \c seqtest3
+    ```
+
+    ```output
+    You are now connected to database "seqtest3" as user "yugabyte".
+    ```
+
+1. Create a sequence baz in seqtest3 and allocate value from sequence baz using nextval. In this scenario, the nextval function returns 5 because the sequence baz shares the same cache entry on the TServer as foo and bar, due to having the same OID.
+
+    ```sql
+    create sequence baz;
+    ```
+
+    ```sql
+    select oid, relname from pg_class where relkind = 'S';
+    ```
+
+    ```output
+      oid  | relname
+    -------+---------
+     16384 | baz
+    (1 row)
+    ```
+
+    ```sql
+    select nextval('baz');
+    ```
+
+    ```output
+    nextval
+    ---------
+           5
+    (1 row)
+    ```
```

Comment:
```suggestion
    ```
    In this scenario, the nextval function returns 5 because the sequence `baz` shares the same cache entry on the TServer as `foo` and `bar`, due to having the same OID.
```